Build/Run-Tests.ps1

```diff
@@ -3,18 +3,25 @@ $packageConfig = [xml](Get-Content .nuget\packages.config)
 $opencover_version = $packageConfig.SelectSingleNode('/packages/package[@id="OpenCover"]').version
 $opencover_console = "packages\OpenCover.$opencover_version\tools\OpenCover.Console.exe"
 
-&$opencover_console `
-    -register:administrator `
-    -returntargetcode `
-    -hideskipped:All `
-    -filter:"+[*]* -[FluentAssertions*]* -[SmartFormat*]* -[nunit*]*" `
-    -excludebyattribute:*.ExcludeFromCodeCoverage* `
-    -excludebyfile:*\*Designer.cs `
-    -output:"OpenCover.GitExtensions.xml" `
-    -target:"nunit3-console.exe" `
-    -targetargs:"$testAssemblies --workers=1 --timeout=90000"
-$testExitCode = $LastExitCode
-Push-AppveyorArtifact "TestResult.xml"
+for ($i=1; $i -le 6; $i++)
```

Comment:
I prefer for this to be an absolute last resort. I've only ever seen one case where it was necessary, and it stemmed from needing to run code that we didn't own and so we couldn't fix bugs in it.

Reply:
Now you've lost me. What are you referring to?
I've added this loop in order to temporarily run the tests multiple times during the 1 hour budget for the AppVeyor VM, (no need to _build_ over an over again). Occasionally, we have hangs and sometimes, we even have `ThreadHelper.JoinableTaskContext == null` which is an absolute logical no-go -- at least in the usage by GE (refer to https://github.com/gitextensions/gitextensions/issues/7382#issuecomment-554677833). If you are sure that no tests do run in parallel, how can `ThreadHelper.JoinableTaskContext` can be `null` during a test?

Reply:
> I've added this loop in order to temporarily run the tests multiple times during the 1 hour budget for the AppVeyor VM, (no need to build over an over again).

This is of course fine for testing. I was just commenting on it as I would object to _merging_ the change.